DEVELOPMENT.md

```diff
@@ -21,21 +21,22 @@ Reformat files:
 make format
 ```
 
-## Releasing a new sdk version
-* Checkout the master branch
-* Either run `make` 
+## Releasing a new SDK version
+* Checkout the `master` branch
+* Either run `make`
 
 or manually run the make targets:
 1. Test
 ```
 make clean test
 ```
-2. Bump version and Build
+2. Bump version and build
 ```
-make bump-patch build 
+make bump-patch build
 ```
-3. Publish the module to pypi
+3. Publish the module to PyPI
 ```
 make publish-module
 ```
-* Finally push the version bump changes to git. Either make a new PR or push to master
\ No newline at end of file
+* Finally push the version bump commit and the generated tag to GitHub. Either
```

Comment:
Er det mulig å pushe tags til PR?

Uansett burde man kanskje dokumentere at man må kjøre `git push --tags` for å dytte ut tags?

Reply:
Vet ikke om taggen blir synlig som en del av PR-en i GitHub, men det blir da å tagge den commiten som hører til PR-en.

Men når jeg tenker meg om så er det kanskje ikke så lurt å lage PR-er for slike commits, for hvis den merges med en merge commit vil man heller at mergen skal bli tagget, og hvis man rebaser vil også taggen bli feil.

Kanskje det er best å oppfordre til å tagge og pushe etter at commiten er merget til `master`?

Reply:
Man kan jo merge PRs med fast-forward, da blir det ingen merge commit :nerd_face: 

Men, det ser ikke ut som tags blir med i PR, så det greieste er vel å bare pushe version bumps rett til master, og så pushe taggen.

Reply:
Noe man kan prøve i fremtiden en gang man har tid til overs er jo å kjøre noe github actions som bumper og tagger for oss ...

Reply:
> Man kan jo merge PRs med fast-forward, da blir det ingen merge commit nerd_face

Ja, men hvis commit-hashen endres av en fast-forward vil taggen forsvinne tror jeg.

> Men, det ser ikke ut som tags blir med i PR, så det greieste er vel å bare pushe version bumps rett til master, og så pushe taggen.

Jepp, jeg oppdaterer.